docs-ref-conceptual/whats-new-overview.md

```diff
@@ -13,37 +13,23 @@ keywords: Azure CLI, new articles, new references, new samples, announcements
 ---
 # What's new in the Azure CLI
 
-This page highlights new features, articles and learning paths for the Azure CLI. Use these quick links to jump straight to the article containing detailed information:
+This page highlights new features, articles and learning paths for the Azure CLI.
 
-- [Reference type and status](#reference-type-and-status)
-- [64-bit install](/cli/azure/install-azure-cli-windows#latest-version)
-- [Tab completion](/cli/azure/install-azure-cli-windows#enable-tab-completion-on-powershell)
-- [WAM sign in](/cli/azure/authenticate-azure-cli#sign-in-with-web-account-manager-wam)
-- [Cheat sheet](cheat-sheet-onboarding.md)
-- [Article index](reference-docs-index.md)
-- [Sample index](samples-index.md)
+## Azure Copilot for Azure CLI
 
-## Reference type and status
+[Microsoft Copilot for Azure](https://techcommunity.microsoft.com/t5/azure-infrastructure-blog/simplify-it-management-with-microsoft-copilot-for-azure-save/ba-p/3981106) (preview) is published! Copilot for Azure is an AI companion that helps you design, operate, optimize, and troubleshoot your cloud infrastructure and services. This benefits Azure CLI users because the knowledge of Azure CLI is built into Copilot.
```

Comment:
Please use the link to the official doc: https://aka.ms/MicrosoftCopilotforAzureDocs
For the description, can you also align with the description from the Copilot for Azure docs page?